modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectRelatedModelsProviderRegistryImpl.java

```diff
@@ -73,8 +75,9 @@ protected void deactivate() {
 		_serviceTrackerMap.close();
 	}
 
-	private String _getKey(String className, String type) {
-		return className + StringPool.POUND + type;
+	private String _getKey(long companyId, String className, String type) {
+		return StringBundler.concat(
+			className, StringPool.POUND, companyId, StringPool.POUND, type);
```

Comment:
@murilostodolni Not sure about the order of the parameters here. The `companyId` is before the `className`, but is placed after `className, StringPool.POUND`

Since we are not in the `service`, I think we can change, in all places, to: `String className, long companyId, String type`

Reply:
Perfect. I just did it.

Thank you